content/communities/maintaining-your-safety-on-github/unblocking-a-user-from-your-organization.md

```diff
@@ -33,6 +33,7 @@ If you selected a specific amount of time to block the user, they will be automa
 
 {% data reusables.profile.access_org %}
 {% data reusables.profile.org_settings %}
+{% data reusables.organizations.moderation-settings %}
 {% data reusables.organizations.block_users %}
```

Comment:
Whoops, just noticed one more thing that we should add to this PR.

Let's also update this reusable since the location has slightly changed. Let's use this:

>Under "Moderation settings"

Instead of this.

>In your organization's Settings sidebar

Let's also update the screenshot. When you've finished, let us know so we can review it again! 😊